.tests/aws-alb-logs/config.yaml

```diff
@@ -0,0 +1,11 @@
+parsers:
```

Comment:
can we add s00 stage parser

`- crowdsecurity/syslog-logs`

Even though it is not really used it "real runtim" there is always a stage s00 parser